README.md

```diff
@@ -4,63 +4,85 @@
 
 # EPAM Indigo projects #
 
-Copyright (c) 2009 to present EPAM Systems
-Apache License version 2.0
+Copyright (c) 2009-2020 EPAM Systems, Inc.
```

Comment:
We decided to write "2009 to present" to not update year each time.

Reply:
Reason of this change is to follow standards.
Finally, we agreed on this change.